pkg/volume/csi/csi_attacher.go

```diff
@@ -286,15 +288,25 @@ func (c *csiAttacher) MountDevice(spec *volume.Spec, devicePath string, deviceMo
 	}
 
 	// Setup
-	if spec == nil {
```

Comment:
This check doesn't seem to be replicated elsewhere. Is spec actually nil in the ephemeral case? I think the code below implies not?

Reply:
The nil spec check is done in call `getSourceFromSpec()` call below

Reply:
Ah. I missed that. looks good to me.